packages/common/src/index.ts

```diff
@@ -1251,6 +1251,8 @@ export function formatValue<T>(format: string, value: T): string | T {
             return `$${value}`;
         case 'gbp':
             return `£${value}`;
+        case 'eur':
+            return `${value}€`;
```

Comment:
This should follow the same convention as USD and GBP of coming before the number

Reply:
From wiki: https://en.wikipedia.org/wiki/Euro_sign
```
 In English, the sign immediately precedes the value (for instance, €10); in most other European languages, it follows the value, usually but not always with an intervening space (for instance, 10 €, 10€).
 ```
It makes sense because the site is in english, but I've personally never seen the € before the value.

Reply:
I couldn't find a counter example, so let's keep it consistent as you sugested